src/NUnitEngine/Addins/addin-tests/VSProjectTests.cs

```diff
@@ -23,20 +23,19 @@
 
 using System;
 using System.IO;
-using System.Collections;
-using NUnit.Engine.Tests.resources;
+using NUnit.Engine.Services.ProjectLoaders;
 using NUnit.Framework;
 
-namespace NUnit.Engine.Services.ProjectLoaders.Tests
+namespace NUnit.Engine.Tests
```

Comment:
Not arbitrary, matches the directory it is in.

On Saturday, April 11, 2015, CharliePoole <notifications@github.com> wrote:

> In src/NUnitEngine/Addins/addin-tests/VSProjectTests.cs
> <https://github.com/nunit/nunit/pull/586#discussion_r28157483>:
>
> >  using NUnit.Framework;
> >
> > -namespace NUnit.Engine.Services.ProjectLoaders.Tests
> > +namespace NUnit.Engine.Tests
>
> Seems arbitrary.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/586/files#r28157483>.
>